### PR TITLE
Add background color to Bubble component

### PR DESCRIPTION
**Pull request by wasilakkf**
screen shot 2018-07-10 at 11 00 43

### CODE REVIEW

**Comment by ElaLukaszewicz** [comment]
add please different versions of color in bubble.jsx - so that it will be visible in style-guide not only in interactive mode

File: src/components/bubble/_bubble-mixin.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@mixin bubbleColor() {
```
Comment (aju) [inline src/components/bubble/_bubble-mixin.scss]: I think this mixin is useless

File: src/components/bubble/Bubble.jsx [hunk 2]
```diff
@@ -7,7 +7,21 @@ export const DIRECTION = {LEFT: 'left', RIGHT: 'right', TOP: 'top', BOTTOM: 'bot
 
 const HORIZONTAL_DIRECTIONS = [DIRECTION.LEFT, DIRECTION.RIGHT];
 
-const Bubble = ({alignment = ALIGNMENT.CENTER, direction, full, children, className, ...props}) => {
+export const COLOR = {
```
Comment (aju) [inline src/components/bubble/Bubble.jsx]: BUBBLE_COLOR

File: src/components/bubble/pages/bubble-interactive.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Bubble, {DIRECTION, ALIGNMENT} from '../Bubble';
+import Bubble, {DIRECTION, ALIGNMENT, COLOR} from '../Bubble';
```
Comment (aju) [inline src/components/bubble/pages/bubble-interactive.jsx]: BUBBLE_COLOR ;)

**Pull request merged**